clang/docs/ReleaseNotes.rst

```diff
@@ -283,6 +283,7 @@ Bug Fixes to C++ Support
   (`#78524 <https://github.com/llvm/llvm-project/issues/78524>`_)
 - Clang no longer instantiates the exception specification of discarded candidate function
   templates when determining the primary template of an explicit specialization.
+- Fixed a crash when MS unqualified dependent base class lookup occurs in an incomplete class.
```

Comment:
```suggestion
- Fixed a crash in Microsoft compatibility  mode where unqualified dependent base class lookup occurs on an incomplete class.
```

Reply:
Is "lookup occurs _on_ an incomplete class" correct here? Lookup is generally performed _in_ some scope (the lookup context).

Reply:
I think "incomplete class context" is a bit too close to the standardese term "[complete class context](http://eel.is/c++draft/class.mem#general-7)", and wouldn't make sense here (since the completeness of the class isn't context dependent here). How about:

> Fixed a crash in Microsoft compatibility mode where unqualified dependent base class lookup searches an incomplete class.

or

> Fixed a crash in Microsoft compatibility mode where unqualified dependent base class lookup searches the bases of an incomplete class.

I prefer the latter.

Reply:
I can live with either, with a mild preference for the latter, feel free to commit with that one.